apps/docs/pages/builderdao-cli/solana_airdrop.md

```diff
@@ -0,0 +1,106 @@
+
+   ```text
+   builderdao admin --help
+
+   Usage: builderdao admin [options] [command]
+
+   admin command
+
+   Options:
+   -h, --help display help for command
+
+   Commands:
+   airdrop [options] Airdrop token to user
+   bs58 [options] helper
+   setAuthor [options] set the author of a migrated tutorial from LV2
+   ```
+
+8. Intuitively, we can use the airdrop command to send tokens. Before we do that, we can use the `builderdao admin airdrop -h` command to see what options we need to pass for the command to execute:
+
+   ```text
+   builderdao admin airdrop --help
+   Usage: builderdao admin airdrop [options]
+
+   Airdrop token to user
+
+   Options:
+   --adminKp <adminKp> Admin KeyPair (bs58 encoded) (env: ADMIN_KP)
+   --address <address> address of the receiver
+   --onlyKafe Airdrop only Kafe Token
+   --onlyBDR Airdrop only BDR Token
+   -h, --help display help for command
+
+   Aidrop info:
+
+   - 1 Kafe Token
+   - 100 BDR Token (frozen)
+
+   Example call to airdrop both Kafe and BDR token:
+   $ builderdao airdrop --adminKp <bs58Secret> --address <bs58Pubkey>
+
+   Example call to airdrop only BDR token:
+   $ builderdao airdrop --adminKp <bs58Secret> --address <bs58Pubkey> --onlyBDR
+
+   Example call to airdrop only Kafe token:
+   $ builderdao airdrop --adminKp <bs58Secret> --address <bs58Pubkey> --onlyKafe
+   ```
+
+9. Here we see that we need to pass a public address for the account we want to send tokens using the option `--address`. Moreover, we need to pass a private key using the `--adminKp` option to authenticate ourselves as an admin. We also see that we have the option to send 1 $KAFE and 100 $BDR at the same time, or to choose one to send by itself.
+
+10. In order to send tokens, you will need the base58 private key of the account you provided to be included as an admin. If you’re using Phantom wallet, follow these steps:
+
+    1. Open the Settings by clicking on the gear on the bottom right of the wallet navigation bar.
+    2. Scroll to the bottom of Settings and click on the button labeled “Export Private Key”.
+    3. Enter your Phantom wallet password.
+    4. Copy the private key (it should go without saying, but please keep this safe, make sure no one is looking at your screen, etc).
+
+11. With your private key in your clipboard, go back to your terminal and issue the following command:
+
+    ```markdown
+    builderdao admin airdrop --adminKp <paste-your-private-key-here>
+    --address <paste-destination-public-address-here>\*\*
+    ✔ signature: <some-signature-hash-if-the-transaction-was-successful>
+    ```
+
+12. If the transaction fails, it will show an error and give a reason at the top of the error message along with a longer error message that you can safely ignore. If this happens, check your private key. And if you can’t resolve it, please contact @Daniel Gamboa or @Moises Beltran.
```

Comment:
It could be useful to show an example error so people know what to look for. It could help us to collect the possible error messages related to airdrops 😅 

For example, the "unsupported sysvar" error message below occurred for me today while testnet is still not operating 100%. Previous errors were related to "missing blockhash".

```text
⠏ Processing AirdropTransaction simulation failed: Error processing Instruction 0: Unsupported sysvar
    Program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL invoke [1]
    Program log: Create
    Program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL consumed 2275 of 1400000 compute units
    Program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL failed: Unsupported sysvar
/Users/rogan/Projects/work/kafe/node_modules/@solana/web3.js/lib/index.cjs.js:6817
      throw new SendTransactionError('failed to send transaction: ' + res.error.message, logs);
            ^

SendTransactionError: failed to send transaction: Transaction simulation failed: Error processing Instruction 0: Unsupported sysvar
    at Connection.sendEncodedTransaction (/Users/rogan/Projects/work/kafe/node_modules/@solana/web3.js/lib/index.cjs.js:6817:13)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Connection.sendRawTransaction (/Users/rogan/Projects/work/kafe/node_modules/@solana/web3.js/lib/index.cjs.js:6772:20)
    at async Connection.sendTransaction (/Users/rogan/Projects/work/kafe/node_modules/@solana/web3.js/lib/index.cjs.js:6762:12)
    at async Object.sendAndConfirmTransaction (/Users/rogan/Projects/work/kafe/node_modules/@solana/web3.js/lib/index.cjs.js:2977:21)
    at async airdrop (/Users/rogan/Projects/work/kafe/dao-programs/programs/tutorial/dist/index.cjs:1865:21)
    at async Command.<anonymous> (/Users/rogan/Projects/work/kafe/packages/cli/dist/cli.cjs:1239:23)
    at async Command.parseAsync (/Users/rogan/Projects/work/kafe/packages/cli/node_modules/commander/lib/command.js:915:5) {
  logs: [
    'Program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL invoke [1]',
    'Program log: Create',
    'Program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL consumed 2275 of 1400000 compute units',
    'Program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL failed: Unsupported sysvar'
  ]
}
```

Reply:
don't think it make sense as the numbers of different error message can be important, this will lead to flood the important information and make the documentation unreadable. If someone meet some error message then he can this contact me using slack.